src/routes/contact-us/+page.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+	import fb_logo from './Facebook_Logo_Primary.png';
+	import ig_logo from './Instagram_Glyph_Gradient.png';
+</script>
+
+<h1>Contact Us</h1>
+
+<p>
+	Email Address: <a href="mailto:kendoinfo@umich.edu"> kendoinfo@umich.edu</a>
+</p>
+
+<h2>Find us on:</h2>
+
+<figure>
+	<a href="https://www.facebook.com/groups/1038072369705184">
+		<img class="social-media" src={fb_logo} alt="Facebook logo" />
+	</a>
+	<a href="https://www.instagram.com/kendoumich/">
+		<img class="social-media" src={ig_logo} alt="Instagram logo" />
+	</a>
+</figure>
+
+<style>
+	.social-media {
+		padding: 1em;
```

Comment:
Is this enough padding for the brand guidelines?

Reply:
Brand guidelines say space between 1 logo to another should be half of the width (or height, since it's 1:1 ratio). That is probably not enough, but I decided to leave that for styling task #21

Reply:
I'm they case you should set it to 50px because the width of the logo is 100px.  Or ideally you would use a variable to store the width of the logis and then use calc() to divide it by 2.

Reply:
I was having a problem where, when I used 50px of padding, the images would "disappear". 
![Screenshot 2024-05-06 121835](https://github.com/kendoclub-at-umich/website/assets/75005450/da66896c-46a0-46dd-87ae-08fe9ee8ab7a)
![Screenshot 2024-05-06 121821](https://github.com/kendoclub-at-umich/website/assets/75005450/2061fe84-cf31-4d52-8a64-acb98d589647)
![Screenshot 2024-05-06 121857](https://github.com/kendoclub-at-umich/website/assets/75005450/1131f62c-27da-4660-97d7-9824b787c49f)

I did some research and learned that the CSS default is [`box-sizing: content-box`](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing), where only content size matters for the width property and padding is added outside of that, which is how I expected it to work. However, it appears that, for some reason, I'm seeing `box-sizing: content-box` behavior where the padding is included with content size in the width property. Not sure why I'm seeing that behavior.

Reply:
Interesting, I wonder where it's getting set.

Reply:
After conversation, we decided to relocate/move some of the social media. Resolved in [5e58e97](https://github.com/kendoclub-at-umich/website/pull/32/commits/5e58e970d6d50441610fd35db7b9caf3c7f235e7)